player/lua/console.lua

```diff
@@ -1779,6 +1780,7 @@ set_active = function (active)
         undefine_key_bindings()
         mp.enable_messages('silent:terminal-default')
         mp.set_property_bool('user-data/mpv/console/open', false)
+        mp.set_property_bool('input-ime', false)
```

Comment:
Wouldn't this disable input-ime even if the user enabled it in their config? It should store the configured value instead.

Reply:
Yes, the third commit essentially makes `--input-ime` useless, except for specifying the mode used upon startup. It should override any property setting by console.lua, and the description should be adjusted to reflect this. (and maybe rename to `--input-force-ime`)